.gitignore

```diff
@@ -13,6 +13,7 @@ config/database.yml
 # Ignore sqlite3 databases
 **/*.sqlite*
 
-# Ignore and env files
+# Ignore general local files
+Guardfile
```

Comment:
Yes, but i needed it to work on this branch. Once we merge the guard PR I will rebase this branch.